### DISCUSSION TITLE
I have a issue with custom css.

### DISCUSSION THREAD

**Discussion by Happie-c** [EddieHubCommunity/support#7906] [General] [answered]
I'm applying effect graidient to text.
I used this code. but not working.

.gradient-text {
  font-size: 36px;
  font-weight: bold;
  background: linear-gradient(to right, #FFD54F, #FFB300); /* Golden gradient from yellow to amber */
  background-clip: text;
  -webkit-text-fill-color: transparent;
  background-clip: text;
  text-fill-color: transparent;
  text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.5); /* Text shadow with 2px horizontal offset, 2px vertical offset, 4px blur radius, and 50% opacity */
}

**Comment by danielleeas** [comment, marked as answer]
that's because broswer interaction.
If you are using chrome, you need to add this line

`-webkit-background-clip: text;`
